scripts/extensions/auto-tagging-widget/src/tsconfig.json

```diff
@@ -1,6 +1,11 @@
 {
     "extends": "superdesk-code-style/tsconfig-strict",
     "compilerOptions": {
-        "outDir": "../dist/src"
-    }
+        "outDir": "../dist/src",
+        "skipLibCheck": true,
```

Comment:
What about installing [types](https://github.com/react-bootstrap/react-bootstrap-typescript)? If that library doesn't work, you could try importing it as `any` so it doesn't affect other libraries.

```
import __OverlayTrigger from 'react-bootstrap/OverlayTrigger';

const OverlayTrigger = __OverlayTrigger as unknown as any;
```

Reply:
will revert those changes, I had some issues but that got fixed when updating typescript